src/outlet/ticketStorage.ts

```diff
@@ -393,7 +397,11 @@ export class TicketStorage {
 			return parts.join('-')
 		}
 
-		return `${decodedToken.eventId}-${decodedToken.ticketIdNumber ?? decodedToken.ticketIdString}`
```

Comment:
Rather than doing this, check for ticketIdNumber or ticketIdString in the decodeTokenData function. 

```
// Add
decodedToken.ticketId = decodedToken.ticketIdString ?? decodedToken.ticketIdNumber;
delete decodedToken.ticketIdString;
delete decodedToken.ticketIdString;
// before this existing code
tokenData = this.propsArrayBufferToHex(decodedToken['ticket'])
```

Reply:
Applied this change. Handling any scenarios when a legacy attestation format is used for ticketId